apps/verticalhome/js/clock.js

```diff
@@ -36,6 +36,74 @@ function Clock() {
   this.timerID = null;
 
   /**
+   * Contains time format (12 Hour or 24 Hour)
+   * @type {string from localization}
+   */
+  this.timeFormat = null;
+
+  /**
+   * Dom Object that contains time
+   * @type {Dom element}
+   */
+  this.clockTime = document.getElementById('clock-time');
+
+  /**
+   * This dom object contains date information
+   * @type {date}
+   */
+  this.date = document.getElementById('date');
+
+  window.addEventListener('timeformatchange', this);
+
+  this.handleEvent = function(e){
+
+    switch(e.type) {
+      case 'timeformatchange':
+        if (!this.l10nready) {
+          return;
+        }
+
+        this.timeFormat = window.navigator.mozHour12 ?
+          navigator.mozL10n.get('shortTimeFormat12') :
+          navigator.mozL10n.get('shortTimeFormat24');
+        this.refreshClock(new Date());
+
+        break;
+    };
+
+  };
+
+  /**
+   * We need to do some refreshing thing after l10n is ready.
+   *
+   * @memberof LockScreen
+   * @this {LockScreen}
+   */
+  this.l10nInit = function ls_l10nInit() {
```

Comment:
as I can see, named functions in this file have prefix 'cl_' meaning 'clock'. Please use this prefix for newly added functions